02.Rmd

```diff
@@ -0,0 +1,24 @@
+# Statistical Learning
+
+**Learning objectives:**
+
+- Define Statistical Learning.
+- Understand the limits on the accuracy of statistical models.
+- Describe the trade-off between more accurate models and more interpretable models.
+- Compare and contrast supervised and unsupervised learning.
+- Compare and contrast regression and classification problems.
+- Measure the goodness of regression model fits.
```

Comment:
I haven't read the chapter yet, but are we learning to measure these things, or learning to recognize the importance of measuring these things (or something along those lines)?

Reply:
It's defining actual measures.